manager.php

```diff
@@ -27,6 +27,11 @@
         	<input type="submit" value="Find Employee" name="findEmployee"></p>		
 	</div>
 	
+
+       	<input type="submit" value="Find Great Customers" name="findCust"></p>	
+	<input type="submit" value="Find Price" name="findExpensiveLoc"></p>
```

Comment:
Find which price? The code says something about average, the html name says 'expensive loc', and when i press it it says 'average price'. This be way more clear about what it will do, and why a manager might want to use it